### ISSUE TITLE
hi,i got a trouble

### ISSUE THREAD

**Issue by lyfree132** [desh2608/crnn-relation-classification#6]
when i use your code run the chinese bio dataset,the accuracy of the validation set and the training set is very high,almost 0.88,but the accuracy of the test data is about 0.23. The distribution of the training and testing data is consistent,Can you know where the problem is? thank you very much.

**Comment by desh2608** [comment]
It is very difficult to point out the fault just from the train and test performance. Some things to look at:

1. Is the testing data significantly different from the training data?
2. What is the class distribution? If the distribution is skewed, you may want to try weighted softmax and use F1 score as the evaluation metric.
3. What is the ratio of training set size to the number of parameters in your model? If it is small, your model may be overfitting.

Almost in any case when a model trains well but does not generalize, having more data and using better regularization helps.
